onboarding/nodejs.md

```diff
@@ -10,13 +10,23 @@ $ npm install --save rollbar
 
 ```javascript
 // include and initialize the rollbar library with your access token
-var rollbar = require("rollbar");
-rollbar.init("{{ server_access_token }}");
-
-// record a generic message and send to rollbar
-rollbar.reportMessage("Hello world!");
+var Rollbar = require("rollbar");
+var rollbar = new Rollbar({
+  accessToken: "{{ server_access_token }}",
+  captureUncaught: true,
+  payload: {
+    environment: 'test'
+  }
+});
 ```
 
```

Comment:
or would you rather keep it as a log only message as it was before - e.g. rollbar.log('Hello world')